docker/docker-compose.yml

```diff
@@ -8,25 +8,25 @@ services:
         volumes:
             - postgres-events:/var/lib/postgresql/data
         expose:
-            - 5432
+            - "5432"
         environment:
             POSTGRES_USER: postgres
             POSTGRES_PASSWORD: ${PW_POSTGRES}
 
     oms-events:
         restart: on-failure
         build:
-            context: ./$PATH_OMS_EVENTS/oms-events
-            dockerfile: ./Dockerfile.dev
-        image: aegee/oms-events:dev
+            context: ./$PATH_OMS_EVENTS/..
+            dockerfile: ./docker/oms-events/Dockerfile.dev
+            image: aegee/oms-events:dev
```

Comment:
how does docker-compose understand if it should use image or build it from dockerfile?

Reply:
it will always build and give that name, it doesn't try to pull that image

seems that indenting is off, I will fix it

Reply:
how do you think we can use the same dockerfile for production (for pulling instead of building)?

Reply:
with the other repo

Reply:
(the one called MyAEGEE)